app/services/s3_query_service.rb

```diff
@@ -136,19 +148,18 @@ def client_s3_files(reload: false)
       resp = client.list_objects_v2({ bucket: bucket_name, max_keys: 1000, prefix: prefix })
       resp_hash = resp.to_h
       objects = parse_objects(resp_hash)
-      parse_continuation(resp_hash, objects)
+      objects += parse_continuation(resp_hash)
```

Comment:
@carolyncole re-wrote this since it was not really appending before. I am not sure why, I would have expected it to append to the objects array we were passing as a parameter but it didn't so I just force the append `+=` here.